.github/workflows/build.yaml

```diff
@@ -67,28 +67,46 @@ jobs:
   build-docker:
     name: Build docker container, tag and upload to registry
     needs: build
-    if: ${{ github.repository == 'armosec/kubescape' }}
```

Comment:
@developer-guy  We added this condition to avoid running the workflow on forked projects

Reply:
okay makes sense, but in the case of working on a forked repo, this might be a blocker thing for contributors to test its feature that might need to build a project. WDYT?